### ISSUE TITLE
COAM\Penalty - Imposed penalty is issued

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#83]
#### Description:

When a CAS staff member clicks 'issue' for an imposed penalty, a few things happen:

1. This creates an entry for the penalty in the Imposed Penalty Grid for the operator and CAS staff
2. This creates a record in eLicensing for the penalty via API
3. An email is sent to the operator (what email?) notifying them they have a penalty
4. The Track payments tab is populated in the Internal imposed penalty workflow

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #82 

#### Notes:

-
